src/Mapping/DiscriminatorColumnMapping.php

```diff
@@ -39,9 +39,9 @@ public function __construct(
      *     type: string,
      *     fieldName: string,
      *     name: string,
-     *     length?: int,
-     *     columnDefinition?: string,
-     *     enumType?: class-string<BackedEnum>,
+     *     length?: int|null,
+     *     columnDefinition?: string|null,
+     *     enumType?: class-string<BackedEnum>|null,
      *     options?: array<string, mixed>,
```

Comment:
I can, but that require a change like
https://github.com/doctrine/orm/pull/11226/commits/d4606f1e335bac0e554e9ecfece6a6b8e201523c

Because if we pass a null option, it will try to set `DiscriminatorColumnMapping->options` to null when it's supposed to always be an array.